app/assets/javascripts/articles_poll.js

```diff
@@ -18,7 +18,7 @@ function handleUpdatedPost(collectionPost, data) {
   var i,
       posts = data.posts;
 
-  collectionPost.data('publishedAt', data.published_at);
+  collectionPost.setAttribute('data-publishedAt', data.published_at);
```

Comment:
Data attributes shouldn't be camel-cased.
```suggestion
  collectionPost.dataset.publishedAt = data.published_at
```